src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java

```diff
@@ -182,8 +189,8 @@ private ResultSet getAllRows(String databaseName, String tableName, Schema schem
 	}
 
 	private Statement createBatchStatement(Connection connection) throws SQLException, InterruptedException {
-		Statement statement = connection.createStatement();
-		statement.setFetchSize(context.getConfig().bootstrapperBatchFetchSize);
+		Statement statement = connection.createStatement(ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY);
+		statement.setFetchSize(Integer.MIN_VALUE);
```

Comment:
caught up with production bugs.  Got back to this, and yeah, I'm not sure how this code ever worked.  when I point it at my db it... doesn't.  But I know for sure that it used to, but hey.

Anyway, I'll merge this.  opened https://github.com/RentTheRunway/maxwell-1/pull/1 for your review (tears down some of the stuff around the old methods), if you wanna merge that I'll merge this.